src/types/businessCase.ts

```diff
@@ -54,7 +54,6 @@ export type RequestDescriptionForm = {
   cmsBenefit: string;
   priorityAlignment: string;
   successIndicators: string;
-  systemIntakeStatus: SystemIntakeStatus;
```

Comment:
Unused. `systemIntakeStatus` is on the top level and not inside of `RequestDescriptionForm`.